pkg/executor/analyze_idx.go

```diff
@@ -88,8 +88,8 @@ func analyzeIndexPushdown(idxExec *AnalyzeIndexExec) *statistics.AnalyzeResults
 		Count:    cnt,
 		Snapshot: idxExec.snapshot,
 	}
-	if idxExec.idxInfo.MVIndex {
-		result.ForMVIndex = true
+	if idxExec.idxInfo.MVIndex || (idxExec.idxInfo.Global && statsVer == statistics.Version2) {
```

Comment:
May I ask what is the current behavior when the stats version is 1?

Reply:
The `ForMVIndex` flag is used in update `stats_meta` table. And for stats version V1, it should update `stats_meta` table directly no matter it already have a record or not.